responses/30-makefile-anatomy.md

```diff
@@ -1,82 +1,93 @@
-Most of our pipelines in R use a `remakefile` file to "orchestrate" the connections among files, functions, and phases. In this issue, we're going to develop a basic understanding of how these files work, starting with the anatomy of a `remakefile`.
+Our `targets` pipelines in R use a `makefile` file to "orchestrate" the connections among files, functions, and phases. In this issue, we're going to develop a basic understanding of how these files work, starting with the anatomy of the `_targets.R` file.
 
-#### Background
-
-A `remakefile` uses the [yaml file format](https://en.wikipedia.org/wiki/YAML). A basic understanding of YAML is important for making use of `remakefiles`, including creating your own or editing existing files. Additionally, several of the tools and workflows common to USGS data science take advantage of a YAML file to do other things too, so you may bump into them elsewhere. YAML ("YAML ain't markup language") files are fairly simple, flexible, and readable, but there is a bit of a learning curve. We're not going to get too far into YAMLs, but reading up on them or finding a good reference for future use might be a good idea.
-
-#### Using remakefiles in data science pipelines
+#### Setting up a `targets` data science pipeline
 
 In addition to phases (which we covered in {{ store.structure_activity_url }}), it is important to decompose high-level concepts (or existing scripts) into thoughtful functions and "targets" that form the building blocks of data processing pipelines. A target is a noun we use to describe a tangible output of function, which is often a file or an R object. Targets can be used as an end-product (like a summary map) or as input _into_ another function to create _another_ target. 
```

Comment:
Yeah that seems unnecessary to repeat. But repetition can help with learning ...

Reply:
I am OK with the duplication since it is such a critical piece of this workflow. I don't think it hurts to read about again in slightly different langauge.